### PR TITLE
deploy: update the default configmap value of helm chart

### PR DESCRIPTION
**Pull request by chenxu1990**
update the configmap from ceph-csi-config-cephfs and ceph-csi-config-rbd to ceph-csi-config

## Related issues ##
#1296 

Fixes: #1296 

## Future concerns ##

List items that are not part of the PR and do not impact it's
functionality, but are work items that can be taken up subsequently.

### CODE REVIEW

File: docs/deploy-cephfs.md [hunk 1]
```diff
@@ -208,6 +208,7 @@ The same requirements from the Kubernetes section apply here, i.e. Kubernetes
 version, privileged flag and shared mounts.
 
 The Helm chart is located in `charts/ceph-csi-cephfs`.
+The default configmap name is `ceph-csi-config-cephfs` instead of `ceph-csi-config`.
```
Comment (Madhu-1) [inline docs/deploy-cephfs.md:211]: @chenxu1990 Thanks for the PR. we can rename the configmap name in helm charts to `ceph-csi-config` and update the upgrade documentation to edit/update the configmap from ceph-csi-config-cephfs` and `ceph-csi-config-rbd` to `ceph-csi-config` so the user will have one configmap for both cephfs and rbd.

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
@Mergifyio rebase

**Comment by chenxu1990** [comment]
@Mergifyio rebase

**Comment by Madhu-1** [comment]
@MergifyIO rebase

**Review by nixpanic — APPROVED** [review]

**Comment by humblec** [comment]
@Madhu-1 ptal

**Pull request merged**